### PR TITLE
Allow use of static or dynamic linked libjnitests

### PR DESCRIPTION
**Pull request by paddy74**
Allow use of a static or dynamically linked libjnitests based of file extension.
Allow functionality for Windows or Linux based file extensions by using the "os.name" property.

Adjusted bracket placement for appropriate code style.

### CODE REVIEW

File: java/in/derros/jni/Utilities.java [hunk 1]
```diff
@@ -21,18 +21,15 @@
...
-            {
+            try {
                 System.load(
                     FileSystems.getDefault()
                             .getPath("./../build/libjnitests.so")  // Dynamic link
                             .normalize().toAbsolutePath().toString());
             }
-            catch (UnsatisfiedLinkError e)
-            {
+            catch (UnsatisfiedLinkError e) {
```
Comment (thefangbear) [inline java/in/derros/jni/Utilities.java]: I don't mean to nitpick, but this is still not properly formatted...

Comment (paddy74) [reply java/in/derros/jni/Utilities.java]: I had thought you were exclusively referencing the bracket placement. Was is spacing around the parentheses you wanted i.e. `catch ( UnsatisfiedLinkError e )`?

Comment (thefangbear) [reply java/in/derros/jni/Utilities.java]: Yes

Comment (thefangbear) [reply java/in/derros/jni/Utilities.java]: No, I meant that turn
```
}
catch (...)
{
```
into
```
} catch (...) {
```

Comment (paddy74) [reply java/in/derros/jni/Utilities.java]: Ah ok. Should be good now (I hope) with [55a32b7](https://github.com/thefangbear/JNI-By-Examples/pull/10/commits/55a32b7a3e1e5121be85426902cb916efbd392a8)

Comment (thefangbear) [reply java/in/derros/jni/Utilities.java]: Looks good to me!

**Comment by thefangbear** [comment]
Thank you!

**Pull request merged**